doni/driver/worker/blazar/device.py

```diff
@@ -51,6 +55,17 @@ def expected_state(cls, hardware: "Hardware", device_dict: "dict") -> dict:
                 "platform_version": "2",
             }
         )
+        for dp_name in hw_props.get("device_profiles", []):
+            if dp_name in device_dict:
+                LOG.warning(
+                    (
+                        f"Device profile {dp_name} already exists as a built-in property, "
+                        "refusing to set on Blazar!"
+                    )
+                )
+                continue
+            # Blazar stores all properties as strings
+            device_dict[dp_name] = "True"
```

Comment:
Should this `unset` the property somewhere?

Reply:
Ideally yes. In practice this is hard, because we do not know the full set of device_profile names. When we are comparing the expected state to the current state, there is an opportunity to delete keys that exist in Blazar but not in our expected state; I propose that's how we handle this.